From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: add tty include to audit.c
Patch-mainline: never, kabi
References: kabi

In 4.4.127, commit 545853704b49f0cdc887c0b2a71382423b5f3f76 (audit: add
tty field to LOGIN event), upstream commit
db0a6fb5d97afe01fd9c47d37c6daa82d4d4001d removed linux/tty.h include
from kernel/audit.c and it made some of the symbols undefined.

Add the include back to satisfy the kABI checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/audit.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/audit.c
+++ b/kernel/audit.c
@@ -64,6 +64,7 @@
 #include <linux/security.h>
 #endif
 #include <linux/freezer.h>
+#include <linux/tty.h>
 #include <linux/pid_namespace.h>
 #include <net/netns/generic.h>
 
